guide/14-deep-learning/unsupervised-ml-model-guide.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>input_features can be simple dataframe too.</p><p>explanatory_rasters - List of the Raster objects, where 2-sized tuples has to be passed for categorical variables as discussed above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>